source/_integrations/huawei_lte.markdown

```diff
@@ -75,8 +75,29 @@ entities varies by device model and firmware version.
 
 {% include integrations/config_flow.md %}
 
-Unauthenticated mode and default list of notification recipient phone
-numbers can be set using the integration's configuration options.
+{% configuration_basic %}
+URL:
+  description: Base URL to the API of the router. Typically, something like `http://192.168.X.1` where `X` is for example `1`, `8`, or `100`. This is the beginning of the location shown in a browser when accessing the router's web interface.
+Verify SSL certificate:
+  description: Whether to verify the SSL certificate of the router when accessing it. Applicable only if the router is accessed via HTTPS, i.e. if the configured URL starts with `https://`.
+Username:
+  description: Username for accessing the router's API. Typically, either `admin`, or left empty (recommended if that works).
+Password:
+  description: Password for accessing the router's API.
+{% endconfiguration_basic %}
+
+{% include integrations/option_flow.md %}
+
+{% configuration_basic %}
+Notification service name:
+  description: Name of the notification service. Used to distinguish between notification services in case there are multiple Huawei LTE devices configured. The name here will be prefixed with `notify.`, i.e. specifying `huawei_lte` will yield `notify.huawei_lte` as the complete service name.
```

Comment:
```suggestion
  description: Name of the notification service. Used to distinguish between notification services in case there are multiple Huawei LTE devices configured. The name here will be prefixed with `notify.`. For example, specifying `huawei_lte` will yield `notify.huawei_lte` as the complete service name.
```